app/views/comments/_form.html.erb

```diff
@@ -92,7 +92,7 @@
         rows="6" 
         cols="40" 
         placeholder="<%= placeholder %>"
-      ><%= location == :edit ? comment.comment : params[:body] %></textarea>
+      ><%= location == :edit ? comment.comment : params[:none] %></textarea>
```

Comment:
Hmm, i guess we need a different solution. What was the original intent of this phrase?

```ruby
location == :edit ? comment.comment : params[:body]
```

Is it to see if the location is "edit" and if it is, display the existing comment text, but if not, to display the value of the `params[:body]` variable? 

I wonder if it makes the assumption that the `params[:body]` variable is corresponding to... the submitted content of a comment, and if we should be checking to ensure it's the correct value in some other way? Remember that this may be used both on initial pageload and also in an AJAX response, which complicates this. 

It could be useful to monitor the network pane of the Dev Tools to see if the partial template being sent from the server to generate this textarea is bearing the correct content?